### DISCUSSION TITLE
Normal Map artifacts

### DISCUSSION THREAD

**Discussion by Kolupsy** [bnpr/Malt#211] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/88381426/147364235-4607fac8-8cf6-44e5-a990-86ada1f7a614.png)
What are these weird artifacts and how can i prevent them (or maybe what have I dont wrong)? I baked this normal map in cycles, saved it as a 16bit TIFF. The normal map works great when used in Eevee or cycles.

**Comment by pragma37** [comment, marked as answer]
You may want to enable the *precomputed tangents* option in the mesh settings.
Otherwise, tangents are calculated *"on the fly"* inside the pixel shader, which works fine on dense meshes, but not so much on low-poly ones.

The reason the option is disabled by default is that retrieving tangents from Blender is very slow.
